R/usage.R

```diff
@@ -112,13 +112,19 @@ fun_info <- function(x) {
         name = as.character(x[[2]]),
         signature = purrr::map_chr(as.list(x[[3]][-1]), as.character)
       )
+    } else if (!is_call(fun, "list", ns = "")) {
```

Comment:
This doesn't seem quite right to me — wouldn't it be safer to re-call `fun_info()` to handle S3 methods etc?

Also I don't understand where a call list `list()` would be coming from — why isn't the test for `is_call("::")`?

Reply:
This is the suggest way for checking for whether a function has a namespace. I agree it's probably not the cleanest.

https://github.com/r-lib/rlang/blob/master/R/call.R#L154-L156

I'm not sure what you mean about S3 methods — a namespace-qualified S3 method?